uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -312,7 +313,7 @@ sub IsCandidateInfoValid {
         }
     }
 
-    # return 0 if found at least one DICOM file without the proper patient name
+    # return > 0 if found at least one DICOM file without the proper patient name
```

Comment:
```suggestion
    # return 0 if found at least one DICOM file without the proper patient name
```

Not sure I understand why > was added here

Reply:
My mistake :) I scanned too quickly.